pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -302,8 +302,8 @@ def root_gpu(self) -> Optional[int]:
 
     @property
     def is_using_torchelastic(self) -> bool:
-        te_flags_passed = "WORLD_SIZE" in os.environ and ("GROUP_RANK" in os.environ or "NODE_RANK" in os.environ)
-        return te_flags_passed
+        required_env_vars = ("RANK", "GROUP_RANK", "LOCAL_RANK")
+        return all(v in os.environ for v in required_env_vars)
 
```

Comment:
this triplet of env variables identifies torch elastic.
hopefully sufficent ? do we know any other clusters that use this combination of names?

Reply:
you could also pick something more obvious from the name like `TORCHELASTIC_RUN_ID`: https://pytorch.org/elastic/0.2.2/distributed.html

Reply:
this env variable in particular is not there in older versions
https://pytorch.org/elastic/0.2.0/distributed.html
but I could include LOCAL_WORLD_SIZE

Reply:
what do you think of making this a static method on the TorchElasticEnvironment class? that way all the environment variables can be colocated in the same place in the code. we can do the same for Slurm

Reply:
yes, makes absolutely sense, let's do it for torchelastic here and then the one for slurm in a follow up. The slurm logic could anyway use some clean up :)